tests/modules/services/window-managers/sway/sway-no-xwayland.nix

```diff
@@ -17,7 +17,7 @@
         ${
           pkgs.writeText "expected" ''
             xwayland disable
-                    ''
```

Comment:
This is a good change, but irrelevant to this PR, so please revert and make a new PR for this

Reply:
So the story behind this change is I used to run `bash format` instead of `./format` because I have nixfmt installed globally and it was faster, but as a result I was using nixfmt 0.5.0 instead of the pinned 0.4.0, so I assumed that a previous PR did not format that file properly when the formatting difference was only due to the update.

nixfmt was updated in https://github.com/nix-community/home-manager/pull/3029 so I'll just drop the commit.